tests/clusterclient.c

```diff
@@ -78,14 +81,16 @@ int main(int argc, char **argv) {
     redisClusterContext *cc = redisClusterContextInit();
     redisClusterSetOptionAddNodes(cc, initnode);
     redisClusterSetOptionConnectTimeout(cc, timeout);
-    redisClusterSetOptionRouteUseSlots(cc);
+    if (use_cluster_slots) {
+        redisClusterSetOptionRouteUseSlots(cc);
+    }
     if (show_events) {
         redisClusterSetEventCallback(cc, eventCallback, NULL);
     }
 
     if (redisClusterConnect2(cc) != REDIS_OK) {
         fprintf(stderr, "Connect error: %s\n", cc->errstr);
-        exit(100);
+        exit(2);
```

Comment:
Maybe we should document this magic number in the comment in the top of the file?

Reply:
Its taken from clientcluster_async, and I guess this is the seconds failure exit() that exits during startup.
The testcase could have been change as an alternative, but I can add some comment here.